### ISSUE TITLE
Error: java.lang.NullPointerException: Cannot invoke "physx.NativeObject.getAddress()" because "value" is null

### ISSUE THREAD

**Issue by mkb2091** [haubna/PhysicsMod#39]
I just tried the new 2.3.5 fabric release with ragdoll enabled. I ran a mob farm and had very low performance as expected due to the large number of bodies in a confined area, but didn't expect a crash. Unsure if at all related, but thought worth mentioning

```
Time: 7/24/21, 10:57 PM
Description: Unexpected error

java.lang.NullPointerException: Cannot invoke "physx.NativeObject.getAddress()" because "value" is null
	at physx.common.PxBoundedData.setData(PxBoundedData.java:121)
	at net.diebuddies.physics.ConvexRigidBodyPX.create(ConvexRigidBodyPX.java:98)
	at net.diebuddies.physics.PhysicsWorldPX.addBlockParticle(PhysicsWorldPX.java:701)
	at net.diebuddies.physics.PhysicsWorldPX.addBlockParticle(PhysicsWorldPX.java:710)
	at net.minecraft.class_761.updatePhysics(class_761.java:6054)
	at net.minecraft.class_761.handler$zbp000$renderLayer(class_761.java:5884)
	at net.minecraft.class_761.method_3251(class_761.java:4327)
	at net.minecraft.class_761.method_22710(class_761.java:1166)
	at net.minecraft.class_757.method_3188(class_757.java:1026)
	at net.minecraft.class_757.method_3192(class_757.java:810)
	at net.minecraft.class_310.method_1523(class_310.java:1112)
	at net.minecraft.class_310.method_1514(class_310.java:728)
	at net.minecraft.client.main.Main.main(Main.java:217)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:567)
	at net.fabricmc.loader.game.MinecraftGameProvider.launch(MinecraftGameProvider.java:234)
	at net.fabricmc.loader.launch.knot.Knot.launch(Knot.java:153)
	at net.fabricmc.loader.launch.knot.KnotClient.main(KnotClient.java:28)
```

**Comment by haubna** [comment]
This happens when a certain non configured mob dies. If you figure out which mob causes it, let me know (future releases will have a fail save approach that will not crash the game)!

**Comment by haubna** [comment]
Fixed in version 2.3.6
